advection_test/tracerAdvection.ipynb

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.                                            cmap='viridis', vmin=0, vmax=1, units='kg', 
</pre><p>unit = kg?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ai2cm/pace/pull/246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
why not?